src/web_interface/static/js/fact_statistics.js

```diff
@@ -82,3 +88,42 @@ function create_horizontal_bar_chart(canvas_id, chart_data, link, value_percenta
 
     return BarChart;
 }
+
+function create_pie_chart(canvas_id, chart_data, link, value_percentage_present_flag) {
+    var ctx = document.getElementById(canvas_id);
```

Comment:
Here's obvious code duplication. Is there a nice javascript way to de-duplicate this?